examples/hi_xiaowen/s0/run_ctc.sh

```diff
@@ -0,0 +1,223 @@
+      tools/make_list.py $trainbase_dir/$x/wav.scp $trainbase_dir/$x/text \
+          $trainbase_dir/$x/wav.dur $trainbase_dir/$x/data.list  \
+          --token_file data/tokens.txt \
+          --lexicon_file data/lexicon.txt
+    fi
+  done
+
+  echo "Start base training ..."
+  mkdir -p $trainbase_exp
+  cmvn_opts=
+  $norm_mean && cmvn_opts="--cmvn_file data/train/global_cmvn"
+  $norm_var && cmvn_opts="$cmvn_opts --norm_var"
+  num_gpus=$(echo $gpus | awk -F ',' '{print NF}')
+  torchrun --standalone --nnodes=1 --nproc_per_node=$num_gpus \
+    wekws/bin/train.py --gpus $gpus \
+      --config $trainbase_config \
+      --train_data $trainbase_dir/train/data.list \
+      --cv_data $trainbase_dir/dev/data.list \
+      --model_dir $trainbase_exp \
+      --num_workers 2 \
+      --ddp.dist_backend nccl \
+      --num_keywords $num_keywords \
+      --min_duration 50 \
+      --seed 666 \
+      $cmvn_opts # \
+      #--checkpoint $trainbase_exp/23.pt
+fi
+
+if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
+  echo "Start training ..."
+  mkdir -p $dir
+  cmvn_opts=
+  $norm_mean && cmvn_opts="--cmvn_file data/train/global_cmvn"
+  $norm_var && cmvn_opts="$cmvn_opts --norm_var"
+  num_gpus=$(echo $gpus | awk -F ',' '{print NF}')
+
+  if $trainbase; then
+    echo "Use the base model you trained as checkpoint: $trainbase_exp/final.pt"
+    checkpoint=$trainbase_exp/final.pt
+  else
+    echo "Use the base model trained with WenetSpeech as checkpoint: mobvoi_kws_transcription/23.pt"
+    if [ ! -d mobvoi_kws_transcription ] ;then
+      git clone https://www.modelscope.cn/datasets/thuduj12/mobvoi_kws_transcription.git
+    fi
+    checkpoint=mobvoi_kws_transcription/23.pt    # this ckpt may not converge well.
+  fi
+
+  torchrun --standalone --nnodes=1 --nproc_per_node=$num_gpus \
+    wekws/bin/train.py --gpus $gpus \
+      --config $config \
+      --train_data data/train/data.list \
+      --cv_data data/dev/data.list \
+      --model_dir $dir \
+      --num_workers 8 \
+      --num_keywords $num_keywords \
+      --min_duration 50 \
+      --seed 666 \
+      $cmvn_opts \
+      ${checkpoint:+--checkpoint $checkpoint}
+fi
+
+if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
+  echo "Do model average, Compute FRR/FAR ..."
+  if $average_model; then
+    python wekws/bin/average_model.py \
+      --dst_model $score_checkpoint \
+      --src_path $dir  \
+      --num ${num_average} \
+      --val_best
+  fi
+  result_dir=$dir/test_$(basename $score_checkpoint)
+  mkdir -p $result_dir
+  stream=true  # we detect keyword online with ctc_prefix_beam_search
+  score_prefix=""
+  if $stream ; then
+    score_prefix=stream_
+  fi
+  python wekws/bin/${score_prefix}score_ctc.py \
+    --config $dir/config.yaml \
+    --test_data data/test/data.list \
+    --gpu 0  \
+    --batch_size 256 \
+    --checkpoint $score_checkpoint \
+    --score_file $result_dir/score.txt  \
+    --num_workers 8  \
+    --keywords 嗨小问,你好问问 \
```

Comment:
it better to replace Chinese char with Latin char in the code

Reply:
Has been done in the latest commit.